### ISSUE TITLE
[Bug] remote play magically not working

### ISSUE THREAD

**Issue by discoveryX64** [redphx/better-xcloud#534]
### Checklist

- [X] I will only use English in my report.
- [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### Questions

- [X] xCloud officially supports my country/region.
- [X] The bug doesn't happen when I disable Better xCloud script.
- [ ] The bug didn't happen in previous Better xCloud version (name which one).

### Device

Other

### Device

Windows pc, android app, kiwi browser

### Operating System

windows 11, android 14

### Android app/Browser Version

edge 130, android app v0.22.0

### Better xCloud Version

26.0.6 (5.9.0), and i have tried 3.5.0 and it still hasnt worked

### Game list

Remote play

### Reproduction Steps

1. Get newest version of better xcloud script (beta or not) and try to remote play
2. Won't work
I have port forwarded all of the ports, I don't have upnp or dmz enabled as its impossible and hard to do respectively on at&t Wi-Fi

**Comment by redphx** [comment]
Did you enable this setting? Everthing works fine for me.

<img width="442" alt="image" src="https://github.com/user-attachments/assets/343d9d2b-516f-4b5e-83b6-db01d6865d90">

**Comment by discoveryX64** [comment]
Yes I did, I have had the setting enabled for a while now

**Comment by discoveryX64** [comment]
By the way what is your timezone so I can get a good time to reply to you?

**Comment by redphx** [comment]
check your ISP #502

**Comment by redphx** [comment]
Try older versions. If it still happens then it's 100% your network.

**Comment by discoveryX64** [comment]
I've tried an older version as said earlier in my ticket, I was suspecting it was my ISP because I've been having this problem with sunshine recently as well. I will close the ticket and if you want to respond I will have it up until it is fixed. I will try bug #502
EDIT:  I dont know if it is my isp or not since the guide doesn't help, the wifi fixed itself, and my xbox boots from using better xcloud. I had an issue like this before and I port fowarded, but I already did that so I don't know.
